k8s/plugin/src/resources/mod.rs

```diff
@@ -34,4 +34,7 @@ pub enum Operations {
     Dump(DumpArgs),
     /// `Upgrade` the deployment.
     Upgrade(UpgradeArgs),
+    /// `Delete` the upgrade resources.
+    #[clap(subcommand)]
+    Delete(Upgrade),
```

Comment:
Please create a new `DeleteResources` enum which contains the Upgrade only for now, and add that type here to allow for future expansion.